JapaneseFilter/sections/specific.txt

```diff
@@ -3,6 +3,8 @@
 !
 ! /banner/banner_ - if it is necessary, make this rule domain specific
 !
+||excite.co.jp/jp/ox/
+www.excite.co.jp##div[id^=gpt_pc_news_]
```

Comment:
missed `"` in CSS selector

Reply:
why with `www`?

Reply:
@Alex-302 Right, however, I found actually the cosmetic rule is not needed if /ox/ is blocked. I'll remove it entirely.
@Sergey-Lyapin Since excite has many subdomains, but no problem to remove www if it's better as the rule won't cause FP. Anyway I'll remove the rule in this case.

Reply:
Or do you prefer to keep the rule anyway even if it doesn't actually hide anything? The rule still hit, just not needed. If so, I'll simply add " and remove www.